simplebot init fails with "AssertionError: plugin is not registered"

I'm following the guide posted in the Delta Chat support forum [here](https://support.delta.chat/t/how-to-set-up-a-delta-bot-in-6-steps/1170/1).

After attempting to initialize the bot it fails with the "AssertionError: plugin is not registered" traceback.

I have reproduced this issue on gentoo, guix and ubuntu systems which have the latest updates.

```
$ simplebot --stdlog debug init $ADDR "$PASSWORD"
2023-11-27 13:57:56,965 - simplebot - DEBUG - registering plugin 'commands'
2023-11-27 13:57:56,965 - simplebot - DEBUG - registering plugin 'filters'
2023-11-27 13:57:56,982 - simplebot - DEBUG - registering plugin 'db'
2023-11-27 13:57:56,982 - simplebot - DEBUG - registered new command '/sub'
2023-11-27 13:57:56,982 - simplebot - DEBUG - registered new command '/unsub'
2023-11-27 13:57:56,983 - simplebot - DEBUG - registered new command '/help'
2023-11-27 13:57:56,983 - simplebot - DEBUG - registered new command '/unban'
2023-11-27 13:57:56,983 - simplebot - DEBUG - registered new command '/ban'
2023-11-27 13:57:56,983 - simplebot - DEBUG - registered new command '/set'
2023-11-27 13:57:56,983 - simplebot - DEBUG - registered new command '/echo'
2023-11-27 13:57:56,983 - simplebot - DEBUG - registered new filter 'simplebot_echo.echo_filter'
2023-11-27 13:58:58,034 - simplebot - INFO - Successfully configured bot@domain.tld
Traceback (most recent call last):
  File "/home/user/.local/bin/simplebot", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/user/.local/lib/python3.11/site-packages/simplebot/main.py", line 24, in main
    parser.main_run(bot=bot, args=args)
  File "/home/user/.local/lib/python3.11/site-packages/simplebot/parser.py", line 111, in main_run
    res = args.subcommand_instance.run(**kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/.local/lib/python3.11/site-packages/simplebot/builtin/cmdline.py", line 152, in run
    success = bot.perform_configure_address(
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/.local/lib/python3.11/site-packages/simplebot/bot.py", line 383, in perform_configure_address
    with self.account.temp_plugin(tracker) as configtracker:
  File "/usr/lib/python3.11/contextlib.py", line 144, in __exit__
    next(self.gen)
  File "/home/user/.local/lib/python3.11/site-packages/deltachat/account.py", line 671, in temp_plugin
    self._pm.unregister(plugin)
  File "/usr/lib/python3.11/site-packages/pluggy/_manager.py", line 206, in unregister
    assert name is not None, "plugin is not registered"
           ^^^^^^^^^^^^^^^^
AssertionError: plugin is not registered
```

--- a/doc/conf.py
+++ b/doc/conf.py
@@ -282,6 +282,8 @@ intersphinx_mapping = {"http://docs.python.org/": None}
 
 # autodoc options
 autodoc_member_order = "bysource"
+
+
 # always document __init__ functions
 def skip(app, what, name, obj, skip, options):
     return skip

--- a/src/simplebot/bot.py
+++ b/src/simplebot/bot.py
@@ -11,7 +11,6 @@ from deltachat.capi import lib
 from deltachat.cutil import from_dc_charpointer
 from deltachat.events import FFIEvent
 from deltachat.message import parse_system_add_remove
-from deltachat.tracker import ConfigureTracker
 
 from .builtin.admin import add_admin, del_admin, get_admins
 from .builtin.cmdline import PluginCmd
@@ -377,19 +376,17 @@ class DeltaBot:
         )
         self.account.update_config(configs)
 
-        tracker = ConfigureTracker(self.account)
-        with self.account.temp_plugin(tracker) as configtracker:
-            self.account.configure()
-            try:
-                configtracker.wait_finish()
-            except configtracker.ConfigureFailed as ex:
-                success = False
-                self.logger.error("Failed to configure: %s", ex)
-            else:
-                set_builtin_avatar(self)
-                success = True
-                self.logger.info("Successfully configured %s", email)
-            return success
+        configtracker = self.account.configure()
+        try:
+            configtracker.wait_finish()
+        except configtracker.ConfigureFailed as ex:
+            success = False
+            self.logger.error("Failed to configure: %s", ex)
+        else:
+            set_builtin_avatar(self)
+            success = True
+            self.logger.info("Successfully configured %s", email)
+        return success
 
     #
     # start/wait/shutdown API

--- a/src/simplebot/builtin/cmdline.py
+++ b/src/simplebot/builtin/cmdline.py
@@ -53,8 +53,8 @@ def deltabot_init_parser(parser) -> None:
         version=simplebot_version,
         help="show program's version number and exit.",
     )
-    path = (
-        lambda p: get_account_path(p)
+    path = lambda p: (
+        get_account_path(p)
         if os.path.exists(get_account_path(p))
         else os.path.abspath(os.path.expanduser(p))
     )

